src/components/grid-aware/ImageHeader/ImageHeader.module.css

```diff
@@ -110,6 +127,11 @@
     margin-bottom: 20px;
     margin-right: 0;
   }
+
+  .descriptionWrapper {
```

Comment:
Nit: Could you keep the selectors within the media query in the same order as they appear outside the media query by moving this above the `.ctaButtonRow`? It makes it easier to scan the file if the selectors are in a consistent order.